QuickMusic/Album.m

```diff
@@ -0,0 +1,86 @@
+//
+//  Album.m
+//  QuickMusic
+//
+//  Created by Oskar Mendel on 1/25/16.
+//  Copyright © 2016 Oskar Mendel. All rights reserved.
+//
+
+#import "Album.h"
+
+@implementation Album {
```

Comment:
Class names has to use prefixes https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html#//apple_ref/doc/uid/20001281-BBCHBFAH

I would like to suggest QM (from QuickMusic)